packages/shared/src/forms/datePicker/DatePickerButton.js

```diff
@@ -10,19 +14,26 @@ import Text from '../../Text';
 import Color from '../../Color';
 
 type Props = {|
-  onPress: () => void,
-  date: Date,
-  iconComponent?: React.Node,
+  +onPress: () => void,
+  +date: ?Date,
+  +formatFunction: ?DateFormatterFunctions,
+  +iconComponent?: React.Node,
 |};
 
 export default function DatePickerButton(props: Props) {
   return (
     <Touchable style={styles.dateTouchBody} onPress={props.onPress}>
       <View style={styles.date}>
         {props.iconComponent}
-        <Text style={styles.dateText}>
-          <Translation passThrough={DateFormatter(props.date).formatToDate()} />
-        </Text>
+        {props.date !== null && (
+          <Text style={styles.dateText}>
+            <Translation
+              passThrough={DateFormatter(props.date)[
+                props.formatFunction || 'formatToDate'
```

Comment:
Same question here: is it not cleaner to use default value for prop `formatFunction`?

Reply:
Definitely, here I wanted to use `defaultProps`, however flow does not like it 🤷‍♂️